<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>536</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QWidget" name="widget_controller" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>32</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#widget_controller
{
	background-color: rgba(0, 255, 0, 100);
}</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton_open_2">
         <property name="text">
          <string>打开视频</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_play_2">
         <property name="text">
          <string>播放</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_pause_2">
         <property name="text">
          <string>暂停</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_stop_2">
         <property name="text">
          <string>停止</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_currenttime_2">
           <property name="text">
            <string>00:00:00</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>/</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_totaltime_2">
           <property name="text">
            <string>00:00:00</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
